app/src/main/res/layout/fragment_routes.xml

```diff
@@ -3,35 +3,52 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:orientation="vertical"
     tools:context=".ui.maps.RouteFragment"
     android:id="@+id/fragment_route_directors_parent">
 
-
-    <Button
-        android:id="@+id/button_next"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="@string/next"
-        android:visibility="invisible"/>
-    <Button
-        android:id="@+id/button_run"
-        android:layout_width="wrap_content"
+    <androidx.appcompat.widget.SearchView
```

Comment:
You should make the search bar clickable area wider.
A good suggestion would be to use add app:iconifiedByDefault="false"